extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -519,6 +520,16 @@ private Properties getSchedulerConfigurationProperties(QuartzSupport quartzSuppo
             if (buildTimeConfig.storeType.isNonManagedTxJobStore()) {
                 props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".nonManagedTXDataSource", dataSource);
             }
+        } else if (buildTimeConfig.storeType.isCustom()) {
+            Optional<QuartzExtensionPointConfig> customStore = buildTimeConfig.customStore();
+            if (!customStore.isPresent()) {
+                LOGGER.warn("Custom job store class property is not set");
+            } else {
```

Comment:
This will never occur because of the build time validation happening in https://github.com/quarkusio/quarkus/blob/e82004d5e20184ac6432bcc7b9358b201b9107f1/extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java#L174 
We can remove the check.

Reply:
removed